templates/channels/types/instagram/claim.haml

```diff
@@ -0,0 +1,136 @@
+-extends "channels/channel_claim_form.html"
+-load i18n compress temba
+
+-block pre-form
+  -blocktrans trimmed with name=brand.name
+    You can connect your Instagram business account to {{name}} in just a few simple steps.
+
+-block form
+  .mt-4.card
+    #fb-guide
+      %ol.steps
+        %li
+          -trans "The page must be linked to your instagram business account, so check how to link"
+          %a{:href => 'https://help.instagram.com/399237934150902'} here
+          first.
+
+        %li
+          -trans "Click on the button below to get started."
+
+        %li
+          -trans "Select the user you want to log in as."
+
+        %li
+          -trans "Select all the pages you want to grant us permissions for."
+
+        %li
+          -trans "Approve the permissions, these are required for us to send and receive messages on your behalf."
+
+        %li
+          -trans "Finally, select the page you want to add as a channel."
+
+    #fb-app-connect.flex.mt-4
+      .button-primary.connect-facebook
+        -trans "Add Instagram Business Account"
+
+    #throbber(style="display:none;")
+      %temba-loading
+
+    %form#claim-form(style="display:none;" method="POST" action="{{ claim_url }}")
+      {% csrf_token %}
+      %input#fb-user-id(type="text" name="fb_user_id")
+      %input#user-access-token(type="text" name="user_access_token")
+      %input#page-id(type="text" name="page_id")
+      %input#page-name(type="text" name="page_name")
+
+  .mt-4.card#fb-channel-options(style="display:none;")
+    %p(style="font-size:1rem;")
+      -trans "Select the page you want to add as a channel:"
```

Comment:
is there a screenshot of this part we can see?

Reply:
![print](https://user-images.githubusercontent.com/30378788/149385803-283d8c88-6aea-417e-88dd-b697f2126aa4.jpeg)